File: Weekly Contest 239/A.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        res = float("INF")
        l, r = start, start
        while l > 0 and nums[l] != target:
            l -= 1
        while r < len(nums)-1 and nums[r] != target:
            r += 1
        if nums[l] == target:
            res = min(res, start - l)
        if nums[r] == target:
            res = min(res, r - start)
        return res

File: Weekly Contest 239/A.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int res = INT_MAX;
        int l = start, r = start;
        while(l > 0 && nums[l] != target) l--;
        while(r < nums.size()-1 && nums[r] != target) r++;
        if(nums[l] == target) res = min(res, start - l);
        if(nums[r] == target) res = min(res, r - start);
        return res;
    }
};